### DISCUSSION TITLE
9.23支付宝续期的1年专业版，还是免费版，无法升级专业版，麻烦处理下谢谢

### DISCUSSION THREAD

**Discussion by leocnx** [getlantern/lantern-forum-cn#891] [Q&A]
邮箱：auok.top@gmail.com
![微信图片_20240925172036](https://github.com/user-attachments/assets/6b0bf099-1907-4765-ba38-b8c63fd7adc7)

**Comment by leocnx** [comment]
@Mike777Khoo

**Reply by leocnx** [reply]
请尽快处理，谢谢
![微信截图_20240925174110](https://github.com/user-attachments/assets/29f7378e-5267-43bc-a47f-1072256ba2c6)

**Comment by MaxMKMax** [comment]
@leocnx 您好. OK 没问题. 我已经帮您提交上去了. 请您放心! 谢谢您支持蓝灯!

**Reply by leocnx** [reply]
怎么还没解决？？？老用户了好不好！！！

**Reply by leocnx** [reply]
@Mike777Khoo
